changes/issue-4846-add-jira-integrations-config

```diff
@@ -0,0 +1 @@
+* Add new Jira integrations configuration support.
```

Comment:
Nit: It was agreed that this initial version will be labeled `alpha`.

Reply:
I haven't seen mention of it in the ticket or the epic, I pushed an update that indicates this but let me know if we have a specific wording we want to use across all mentions of this feature.